lib/src/widgets/release_dialog.dart

```diff
@@ -69,19 +95,23 @@ class _ReleaseDialogState extends State<ReleaseDialog> {
         ElevatedButton(
           onPressed: _selected != null
               ? () {
-                  widget.onInstall(_selected!);
-                  Navigator.of(context).pop();
+                  showConfirmationDialog(
+                    context,
+                    title: '$action ${_selected?.name}?',
```

Comment:
It's safer to pass the arguments for translation instead of formatting the string by hand to let translations control the word order. Also, the basic form of "upgrade" vs. "to upgrade something" can be different in some languages.

Reply:
Sorry to drop in again, but the simplest way to get the close button without any UI weirdness is to use YaruDialogTitle as the title prop of dialogs and set the titlepadding to Edgeinset.zero

Reply:
I changed it back to the simpler `AlertDialog` because I'm not using any of the additional features that `YaruAlertDialog` provides any longer. As @Feichtmeier suspected I didn't like the way the title ran into the close button in the top right corner - thanks for the helpful suggestion though!
Concerning the translation, you're totally right @jpnurmi - that's what I did for the other strings, I just forgot to remove the `title` argument, which isn't currently used in the dialog.

Reply:
I've removed the `title` argument for now and will improve the design of the dialogs in a separate PR.

Reply:
great! small steps for the win :wink: